docs/tutorials/Astro/deploying-hybrid-projects.md

```diff
@@ -0,0 +1,286 @@
+   - Uncomment `excludeRequestHeaders: ['host']` to prevent hostname conflicts
+
+
+## Deploying with Apostrophe Hosting
+
+Apostrophe offers a straightforward hosting solution specifically designed for ApostropheCMS projects, including those with Astro frontends.
+
+### Benefits
+
+- Zero configuration for ApostropheCMS + Astro integration
+- Database provisioning and management handled automatically
+- Built-in asset storage and delivery
+- SSL certificate management
+- Automatic backups
+- Security updates
+
+### Getting Started with Apostrophe Hosting
+
+Contact the Apostrophe team through their [website](https://apostrophecms.com/contact-us) to set up your hosting. The team will provide detailed instructions for connecting your repository for automatic deployment.
+
+## Split Deployment (Separate Backend and Frontend)
+
+For more control or to leverage specific platform features, you can deploy the backend and frontend separately.
+
+### Backend (ApostropheCMS) Deployment
+
+Your ApostropheCMS backend requires:
+
+- Node.js environment (v18 or later recommended)
+- MongoDB database connection
+- Asset storage solution (S3 or equivalent cloud storage)
+
+#### Common Backend Hosting Options
+
+1. **Traditional VPS/Dedicated Servers** (DigitalOcean, Linode, AWS EC2)
+   - Complete control over the environment
+   - Requires server management knowledge
+   - Good for high-performance requirements
+
+2. **Platform as a Service** (Heroku, Render, Railway)
+   - Simpler deployment with less configuration
+   - Often includes easy database integration
+   - Automatic scaling options
+
+#### Example: Deploying to Render
+
+1. Create a new Web Service in Render
+2. Connect your GitHub repository
+3. Configure build settings:
+   - Root Directory: `backend`
+   - Build Command: `npm run build`
+   - Start Command: `npm run serve`
+4. Set environment variables:
+   ```
+   NODE_ENV=production
+   APOS_MONGODB_URI=your_mongodb_connection_string
+   APOS_EXTERNAL_FRONT_KEY=your_shared_secret_key
+   APOS_S3_BUCKET=your-bucket-name
+   APOS_S3_SECRET=your-s3-secret
+   APOS_S3_KEY=your-s3-key
+   APOS_S3_REGION=your-chosen-region
+   ```
+There are several guides for other [deployment options](/guide/hosting.html) and configuring [storage services](/cookbook/using-s3-storage.html) in the main ApostropheCMS documentation.
+
+### Frontend (Astro) Deployment
+
+Your Astro frontend can be deployed to any service that supports SSR (Server-Side Rendering). Depending on the hosting provider you may also need to make changes to your `astro.config.mjs` file. The [Astro.build](https://docs.astro.build/en/guides/deploy/#deployment-guides) site has a number of guides for deployment. The only extra consideration is that we are deploying a monorepo, so you need to take the extra steps to identify the `frontend' folder as the root for your Astro deployment.
+
+#### Common Frontend Hosting Options
```

Comment:
I would put our managed hosting right on this list.

Reply:
Done.

Reply:
It's not on the list though still?

Reply:
To cite benefits, as you've done for the others:

- Hosts the combined Astro + ApostropheCMS monorepo in one step
- Zero latency when Astro communicates with ApostropheCMS
- Configures MongoDB and S3 automatically
- Provides `APOS_EXTERNAL_FRONTEND_KEY` automatically